tests/multi_objective/samplers_tests/test_samplers.py

```diff
@@ -0,0 +1,169 @@
+from typing import Callable
+from typing import Sequence
+
+import numpy as np
+import pytest
+
+import optuna
+from optuna.distributions import CategoricalDistribution
+from optuna.distributions import DiscreteUniformDistribution
+from optuna.distributions import IntUniformDistribution
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna import multi_objective
+from optuna.multi_objective.samplers import BaseMultiObjectiveSampler
+from optuna.multi_objective.trial import CategoricalChoiceType
+
+parametrize_sampler = pytest.mark.parametrize(
+    "sampler_class", [optuna.multi_objective.samplers.RandomMultiObjectiveSampler,],
+)
+
+
```

Comment:
Sorry I could not imagine the way you proposed well. > using a mock class of the BaseDistribution class is a simpler
Instead, I merged these tests into one test function since there was a lot of redundant logic (https://github.com/optuna/optuna/pull/1054/commits/8387012371ecbf9b3dc3151cc55d70886e95bdef).
What do you think of the current code?

Reply:
Sorry, this was my mistake and I thought the distribution classes are responsible to ensure the domains of sampling at the time I added the review comment.
Please ignore this.